HackerRank Algos/WarmUp Challenges/Sales_by_Match_1-25-2021.js

```diff
@@ -60,4 +60,6 @@ function sockMerchant(arr) {
   }
   return(pairs);
 
-}
\ No newline at end of file
+}
+
+//This is the Algo i need help with
```

Comment:
@ParkerLBrown I would simplify this whole thing. use the array "reduce" function to group by value, then you can what you did with Math.floor and divide by two.